bindings/python/justfile

```diff
@@ -55,9 +55,9 @@ clean:
 stub-gen: setup
     @echo "{{ BOLD }}--- Generating Python type stubs ---{{ NORMAL }}"
     @cargo run --quiet --manifest-path=../../dev/Cargo.toml -- generate -l python
-    @cargo run --quiet --bin stub_gen
+    @bash -c 'PY_PREFIX=$($(uv python find) -c "import sys; print(sys.base_prefix)") && PYO3_PYTHON=$(uv python find) PYTHONHOME="$PY_PREFIX" LIBRARY_PATH="$PY_PREFIX/lib" DYLD_LIBRARY_PATH="$PY_PREFIX/lib" cargo run --quiet --bin stub_gen'
```

Comment:
This seems not the correct fix to me. Can we use `uv run cargo run --quiet --bin stub_gen` instead to get the correct python?

Reply:
let me check